### PR TITLE
Refac prog

### PR DESCRIPTION
**Pull request by albertmink**
Note for reviewer: Review single commits rather then the entire diff.

### CODE REVIEW

**Review by wurzka — APPROVED** [review]

File: src/z_generate_repo.prog.abap [hunk 1]
```diff
@@ -384,8 +367,7 @@ CLASS lcl_generator IMPLEMENTATION.
       ELSE.
         add_file_to_zip( i_stringtab_content = schema_content
                          i_file_name         = |{ object_type_folder_name }/{ to_lower( objecttype ) }-v{ format_version }.json| ##NO_TEXT
-                         i_error_text        = |The schema for interface { intfname } could not be created. Error when transforming schema content from string to xstring| ##NO_TEXT
-                         ).
+                         i_error_text        = |The schema for interface { intfname } could not be created. Error when transforming schema content from string to xstring| ##NO_TEXT ).
```
Comment (albertmink) [inline src/z_generate_repo.prog.abap:370]:
```suggestion
                         i_error_text        = |The schema for interface { intfname } could not be created. Error when transforming schema content from string to xstring| ).
```

File: src/z_generate_repo.prog.abap [hunk 2]
```diff
@@ -384,8 +367,7 @@ CLASS lcl_generator IMPLEMENTATION.
       ELSE.
         add_file_to_zip( i_stringtab_content = schema_content
                          i_file_name         = |{ object_type_folder_name }/{ to_lower( objecttype ) }-v{ format_version }.json| ##NO_TEXT
-                         i_error_text        = |The schema for interface { intfname } could not be created. Error when transforming schema content from string to xstring| ##NO_TEXT
-                         ).
+                         i_error_text        = |The schema for interface { intfname } could not be created. Error when transforming schema content from string to xstring| ).
```
Comment (albertmink) [inline src/z_generate_repo.prog.abap:370]:
```suggestion
                         i_error_text        = |The schema for interface { intfname } could not be created. Error when transforming schema content from string to xstring| ) ##NO_TEXT.
```

**Comment by albertmink** [comment]
Tested with INTF and example.

**Comment by albertmink** [comment]
@huber-nicolas can you do manuell testing? The unit-tests are green and I could generate a folder for INTF with example and readme,  but I guess there is more to test. We don't want to have regressions in this report.

**Review by huber-nicolas — APPROVED** [review]
I tested your change by generating the repo folder for a FUGR. Looks good to me

**Pull request merged**